### ISSUE TITLE
Installer failing because of wine version name

### ISSUE THREAD

**Issue by AccountOneOff** [lutris/lutris#1404]
Trying to fix [THUG2: THUG Pro Mod version](https://lutris.net/games/tony-hawks-underground-2/) and found this bug. I go to test the installer, cancel install because of errors, make edits, rerun installer, and this gets spit out:

```
wine: relocation error: wine: symbol wine_get_patches version WINE_1.0 not defined in file libwine.so.1 with link time reference
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/installerwindow.py", line 261, in on_installer_selected
    self.prepare_install(self.installer_choice)
  File "/usr/lib/python3/dist-packages/lutris/gui/installerwindow.py", line 267, in prepare_install
    self.interpreter = interpreter.ScriptInterpreter(script, self)
  File "/usr/lib/python3/dist-packages/lutris/installer/interpreter.py", line 124, in __init__
    self.target_path = self.get_default_target()
  File "/usr/lib/python3/dist-packages/lutris/installer/interpreter.py", line 136, in get_default_target
    config = LutrisConfig(runner_slug=self.runner)
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 168, in __init__
    self.update_cascaded_config()
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 198, in update_cascaded_config
    self.system_config.update(self.get_defaults('system'))
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 269, in get_defaults
    options_dict = self.options_as_dict(options_type)
  File "/usr/lib/python3/dist-packages/lutris/config.py", line 280, in options_as_dict
    if self.runner_slug
  File "/usr/lib/python3/dist-packages/lutris/sysoptions.py", line 353, in with_runner_overrides
    runner = runner()
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 193, in __init__
    'default': get_default_version(),
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 166, in get_default_version
    installed_versions = get_wine_versions()
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 124, in get_wine_versions
    version = get_system_wine_version(WINE_PATHS[build])
  File "/usr/lib/python3/dist-packages/lutris/util/wine.py", line 182, in get_system_wine_version
    version = subprocess.check_output([wine_path, "--version"]).decode().strip()
  File "/usr/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['wine', '--version']' returned non-zero exit status 127.
```

End up having close Lutris in order to test the installer again.

**Comment by AccountOneOff** [comment]
It has to do with the wine version used I think. Both `3.6-i386` and `3.16-i386` fail:

https://paste.ubuntu.com/p/Q7F9VMQgGJ/

Switched to staging-3.16-i386 and no problems.

**Comment by strycore** [comment]
Fixed by 81fd71eefc4400428ac215aa4897e8a167eea70c
